fix: incomplete sentence in array page (#28369)
mdn/content: files/en-us/web/javascript/reference/global_objects/array/index.md

console.log(fruits.length); // 6
```

> Increasing the {{jsxref("Array/length", "length")}} extends the array by adding empty slots, i.e. no new elements are created.

Reviewer: ```suggestion
Increasing the {{jsxref("Array/length", "length")}} extends the array by adding empty slots without creating any new elements — not even `undefined`.
```
